### DISCUSSION TITLE
100 GB Query

### DISCUSSION THREAD

**Discussion by djouallah** [duckdb/duckdb#4321] [General]
shared this in Linkedin, and people are amazed that you can run SQL Query on a 100 GB dataset just using DuckDB, great Job
https://www.linkedin.com/posts/mimounedjouallah_aggregate-100-gb-using-powerquery-and-duckdb-activity-6962284003190087680-w6_X?utm_source=linkedin_share&utm_medium=member_desktop_web

Just curious if any one tried a 1 TB Query ?

**Comment by Alex-Monahan** [comment]
Biggest DuckDB file I've worked with so far was about 400GB.
